src/main/java/se/sundsvall/billingdatacollector/integration/opene/kundfakturaformular/model/ExternFaktura.java

```diff
@@ -0,0 +1,37 @@
+package se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model;
+
+import se.sundsvall.billingdatacollector.integration.opene.util.annotation.XPath;
+
+public record ExternFaktura(
+
+	@XPath("/FlowInstance/Header/Flow/FamilyID") String familyId,
```

Comment:
Why are you using mixed swedish and english?
Why are you using swedish at all?

Reply:
Open E